extensions/wikia/PhalanxII/hooks/PhalanxUserBlock.class.php

```diff
@@ -9,7 +9,7 @@
 class PhalanxUserBlock extends WikiaObject {
 	private static $typeBlock = null;
 	private static $checkEmail = false;
-	function __construct(){
+	function __construct() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method "__construct" to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)